src/features/discord/messages/services/discord-message-author-service.ts

```diff
@@ -4,10 +4,13 @@ import { addDiscordDevPrefix } from "../../functions/add-discord-dev-prefix";
 import { DiscordAuthorService } from "../../users/services/discord-author-service";
 import { IDiscordMessageResponse } from "../interfaces/discord-message-response";
 import { AnyDiscordMessage } from "../types/any-discord-message";
+import { LoggerService } from "../../../logger/services/logger-service";
 
 export class DiscordMessageAuthorService {
   private static _instance: DiscordMessageAuthorService;
 
+  private readonly _loggerService = LoggerService.getInstance();
```

Comment:
If you want to follow the style, this props should be after the get instance.
This is strange that ESLint did not triggered this because static fields should be put together.
Will look into it